pkg/foundation/log/ctxlogger.go

```diff
@@ -49,35 +50,26 @@ func Nop() CtxLogger {
 	return CtxLogger{Logger: zerolog.Nop()}
 }
 
-// Prod returns a production logger. Output is formatted as JSON, minimum level
-// is set to INFO.
-func Prod() CtxLogger {
+// InitLogger returns a logger initialized with the wanted level and format
+func InitLogger(level zerolog.Level, format string) CtxLogger {
 	w := zerolog.NewConsoleWriter()
 	w.TimeFormat = "2006-01-02T15:04:05+00:00"
-	w.Out = os.Stderr
-
-	zlogger := zerolog.New(w).
-		With().
-		Timestamp().
-		Stack().
-		Logger().
-		Level(zerolog.InfoLevel)
-
-	return New(zlogger)
-}
-
-// Dev returns a development logger. Output is human readable, minimum level is
-// set to DEBUG.
-func Dev() CtxLogger {
-	w := zerolog.NewConsoleWriter()
-	w.TimeFormat = "2006-01-02T15:04:05+00:00"
-
-	zlogger := zerolog.New(w).
-		With().
-		Timestamp().
-		Stack().
-		Logger().
-		Level(zerolog.DebugLevel)
+	var zlogger zerolog.Logger
+	if strings.EqualFold(format, "cli") {
+		zlogger = zerolog.New(w).
```

Comment:
IIUC, zerlogger's ConsoleWriter parses JSON input and prints it in color. With that, shouldn't we swap these if branches?